apps/dashboard/components/Layout/index.tsx

```diff
@@ -52,7 +54,13 @@ export default function Layout(props: Props) {
 
   React.useEffect(() => {
     setMounted(true);
-  }, []);
+    if (session?.user?.language) {
```

Comment:
I think, i18next has a plugin for automatic locale detection. use i18next-browser-languagedetector